src/client/App.jsx

```diff
@@ -1,11 +1,13 @@
 import './App.css';
 
 import Login from './components/Login';
+import Register from './components/Registration';
 
 export default function App() {
   return (
     <div className="App">
       <Login />
+      <Register />
```

Comment:
Has the component now been changed to Register? That works for me.

Reply:
Missed that, will change Register to Registration to match the exported component